src/components/OSCALMetadata.js

```diff
@@ -33,12 +35,53 @@ const useStyles = makeStyles((theme) => ({
 // Returns a string with a locality-sensitive representation of this date
 const formatDate = (isoDate) => new Date(isoDate).toLocaleString();
 
-export default function OSCALControlGuidance(props) {
+export default function OSCALMetadata(props) {
   const classes = useStyles();
+
+  /* A JSON formatted variable so only one variable needs to be passed around between
+   *  functions that deals with the modification of metadata fields.
+   *
+   *  For each top-level field, we have a value sub-field which is an array of two
```

Comment:
This is a bit confusing.  We may need to revisit this approach after addressing other changes.

Reply:
I'm going to get rid of this approach, as there are now very few fields in the object.